test/cql-pytest/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest
+cassandra-driver
+scylla-driver
```

Comment:
The goal of `requirements.txt` is to write the list of packages needed to run all the tests. You are right because there is no need to install the two drives together, but there is no nice way (other than to create two different files) to do it.